components-core/src/main/java/com/airwallex/android/core/Airwallex.kt

```diff
@@ -626,6 +627,35 @@ class Airwallex internal constructor(
         )
     }
 
+    /**
+     * Checkout the payment by paymentType and session
+     *
+     * @param session a [AirwallexSession] used to present the Checkout flow, required.
+     * @param paymentType a [String] representing one of the redirect payment type names, required.
```

Comment:
This can refer to API reference: https://www.airwallex.com/docs/api#/Payment_Acceptance/Config/_api_v1_pa_config_payment_method_types/get JSON Object field: items.name

Reply:
OK